src/app/shared/picker/icon-picker/icon-picker.component.html

```diff
@@ -10,25 +10,15 @@
 </div>
 <div class="tab-content-container">
   <div class="tab-content align-self-start justify-content-center">
-    <div *ngFor="let iconOption of filteredIcons | pageSlice:page:ICONS_ON_PAGE" class="text-center rounded icon"
-         [class.selected]="iconOption === selected"
-         [class.active]="iconOption === icon"
-         (mouseenter)="preview(iconOption)"
-         (mouseleave)="preview(null)"
-         (click)="select(iconOption)">
-      <i id="{{iconId(iconOption)}}" class="fa-fw {{ iconOption }}" aria-hidden="true"></i>
-    </div>
-  </div>
-  <div class="text-center text-muted px-0 my-1">
-    <strong>{{ pageStart() }}&mdash;{{ pageEnd() }}</strong>
-    &nbsp;<span i18n="@@count.of">of</span>&nbsp;
-    {{ iconsCount() }}
-    <a *ngIf="canActivatePage(page - 1)" (click)="selectPage(page - 1)"><i
-      class="clickable fas fa-angle-double-left"></i></a>
-    <i *ngIf="!canActivatePage(page - 1)" class="fas fa-angle-double-left"></i>
-    <a (click)="activatePageWithSelectedIcon()"><i class="clickable far fa-dot-circle mx-1"></i></a>
-    <a *ngIf="canActivatePage(page + 1)" (click)="selectPage(page + 1)"><i
-      class="clickable fas fa-angle-double-right"></i></a>
-    <i *ngIf="!canActivatePage(page + 1)" class="fas fa-angle-double-right"></i>
+    <i *ngFor="let iconOption of filteredIcons" class="fa-fw {{ iconOption }} text-center rounded icon"
+       [class.selected]="iconOption === selected"
+       [class.active]="iconOption === icon"
+       (mouseenter)="preview(iconOption)"
+       (mouseleave)="preview(null)"
+       (click)="select(iconOption)"
+       id="{{iconId(iconOption)}}"
```

Comment:
This is used inside `ngFor` iterating over a huge array of items so I would highly recommend using a pipe here.